gpii/node_modules/flowManager/src/FlowManagerUtilities.js

```diff
@@ -20,31 +21,26 @@
 
     var fluid = require("infusion"),
         $ = fluid.registerNamespace("jQuery"),
-        gpii = fluid.registerNamespace("gpii"),
-        kettle = fluid.registerNamespace("kettle");
+        gpii = fluid.registerNamespace("gpii");
 
     require("./PrivacyFilter.js");
 
-    fluid.registerNamespace("gpii.request.flowManager.privacy");
 
-    gpii.request.flowManager.logAndNotify = function (msg, event, callback) {
+    gpii.flowManager.logAndNotify = function (msg, event, callback) {
         return function (data) {
             fluid.log(msg, data);
             event.fire(callback ? callback(data) : data);
         };
     };
 
-    gpii.request.flowManager.getPreferences = function (preferencesDataSource, request, event, userToken) {
-        fluid.log("gpii.request.flowManager.getPreferences called - fetching preferences");
+    gpii.flowManager.getPreferences = function (preferencesDataSource, request, event, userToken) {
+        fluid.log("gpii.flowManager.getPreferences called - fetching preferences from URL " + preferencesDataSource.options.url);
```

Comment:
I though the was preferable when using fluid.log - but perhaps, that doesn't apply when we're relatively certain that the variable refernece is a string